src/foraging_gui/Visualization.py

```diff
@@ -57,7 +54,7 @@ def _Update(self,GeneratedTrials=None,Channel=None):
         self.B_RewardOutcomeTime=GeneratedTrials.B_RewardOutcomeTime
         self.B_LaserOnTrial=np.array(GeneratedTrials.B_LaserOnTrial)
         self.B_AutoWaterTrial=GeneratedTrials.B_AutoWaterTrial
-        self.MarchingType=GeneratedTrials.TP_MartchingType
+        self.MarchingType=GeneratedTrials.TP_MartchingType = ['log ratio']
```

Comment:
GeneratedTrials.TP_MartchingType  was being set to ['log ratio'] which correlated to the current item set in the [MartchingType combo box](https://github.com/AllenNeuralDynamics/dynamic-foraging-task/blob/4de385abc00b4b1c6856e6891358dce07ae3f398/src/foraging_gui/ForagingGUI.ui#L1622C25-L1622C72). Since I got rid of the combo box, I needed to set both the MarchingType and the GeneratedTrials.TP_MartchingType to ['log ratio']. I don't see where GeneratedTrials.TP_MartchingType is actually being assigned in the code or used but I wasn't sure if the downstream processing needed TP_MartchingType so I left it in.